### ISSUE TITLE
RAF-DB结果复现

### ISSUE THREAD

**Issue by wjy110** [birdwcp/QCS#4]
您好，使用QCS模型训练raf-db数据集时，想要达到93.02的准确率，是使用AffectNet-8的checkpoint还是使用93.02后面的链接里面的checkpiont。使用AffectNet-8的checkpoint我的准确率在训练120轮后只能达到92.8。

**Comment by wjy110** [comment]
您好，使用QCS模型训练raf-db数据集时，想要达到93.02的准确率，是使用AffectNet-8的checkpoint还是使用93.02后面的链接里面的checkpiont。使用AffectNet-8的checkpoint我的准确率在训练120轮后只能达到92.8。

**Comment by birdwcp** [comment]
> 您好，使用QCS模型训练raf-db数据集时，想要达到93.02的准确率，是使用AffectNet-8的checkpoint还是使用93.02后面的链接里面的checkpiont。使用AffectNet-8的checkpoint我的准确率在训练120轮后只能达到92.8。

We fine-tuned QCS on the pre-trained model QCS-Affect-8(64.3%). The training details are provided in the Appendix of the Extended version (https://arxiv.org/abs/2411.01988). Each experiment was repeated five times, and 93.02% represents the maximum accuracy achieved. You could try training it a few more times.

**Comment by birdwcp** [comment]
> 您好，使用QCS模型训练raf-db数据集时，想要达到93.02的准确率，是使用AffectNet-8的checkpoint还是使用93.02后面的链接里面的checkpiont。使用AffectNet-8的checkpoint我的准确率在训练120轮后只能达到92.8。

We fine-tuned QCS-RAF-DB(93.02%) on the pre-trained model QCS-Affect-8(64.3%). The training details are provided in the Appendix of the Extended version (https://arxiv.org/abs/2411.01988). Each experiment was repeated five times, and 93.02% represents the maximum accuracy achieved. You could try training it a few more times.
